crates/sui-graphql-rpc/src/server/graphiql_server.rs

```diff
@@ -39,6 +39,7 @@ async fn start_graphiql_server_impl(
     // Add GraphiQL IDE handler on GET request to `/`` endpoint
     let server = server_builder
         .route("/", axum::routing::get(graphiql))
+        .route("/graphql", axum::routing::get(graphiql))
```

Comment:
I suspect that this only serves the GraphiQL web UI from `/graphql` and not the service itself (unless you've tested this then feel free to ignore me!) let me double check.

Reply:
Here it is:

https://github.com/MystenLabs/sui/blob/42c66fce68623abecf3a09618d83288bae2fd047/crates/sui-graphql-rpc/src/server/builder.rs#L109

Reply:
you are right, thank you

i addred the route to the service as well as updated the test plan

```
curl -s -X POST -H "Content-Type: application/json" --data '{ "query": "query { chainIdentifier }", "variables": { "userId": "123" } }' http://localhost:8001/graphql | jq
{
  "data": {
    "chainIdentifier": "4c78adac"
  }
}
```